### PR TITLE
Fixes #777 - Downgrade modernizr to 2.8.3

### LINKED ISSUE #777: Upgrade bootstrap to 3.4.0 in `wicket-8.x` branch
As discussed in #773

### PR DESCRIPTION
**Pull request by tgoetz**
See discussion in #777

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
     /**
      * The version of modernizr
      */
-    String MODERNIZR_VERSION = "2.8.3-1";
+    String MODERNIZR_VERSION = "2.8.3";
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java:22]: Same should be done in pom.xml: https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/80fa4d865f89eee8b606291d5757b442c109a904/pom.xml#L64

**Comment by mweirauch** [comment]
Just a question from somebody lurking around: Is it necessary to create 3 PRs? Weren't fixes on the "main" branch ported/merged to the respective other branches anyway?

**Comment by martin-g** [comment]
Yes, they would be cherry-picked. PRs makes it easier for me - I just click around.

**Comment by martin-g** [comment]
Thank you, @tgoetz !

**Pull request merged**